cylc/flow/cycling/integer.py

```diff
@@ -588,6 +588,22 @@ def __lt__(self, other: 'IntegerSequence') -> bool:
                     return True
         return False
 
+    def __repr__(self) -> str:
+        """
+        >>> IntegerSequence('R6/3/P1', '2')
+        <IntegerSequence start=3, stop=8, step=P1, self.p_context_start=2,
+        i_offset=P0>
```

Comment:
Silly little diff to get the coverage to 100%:

```suggestion
        i_offset=P0>
        
        >>> IntegerSequence('R6/P1/3', '2', '4')
        <IntegerSequence start=2, stop=3, step=P1, self.p_context_stop=4,
        i_offset=P0>
```